etc/pam_pkcs11.conf.example.in

```diff
@@ -45,7 +45,7 @@ pam_pkcs11 {
   # will bypass pam_pkcs11 if a token was not used to login (The basic idea is
   # you always unlock the screen saver with the same mechanism you used to
   # login).
-  screen_savers = gnome-screensaver, kde4-kscreensaver, kscreensaver, xscreensaver;
+  screen_savers = mate-screensaver, gnome-screensaver, kde4-kscreensaver, kscreensaver, xscreensaver;
```

Comment:
Maybe you can merge this commit in the previous one?

Reply:
Thanks for pointing this out. Actually, it seems reasonable to move this one to another PR: it's just an update of the screensavers list and has nothing to do with the main thing: `card_only`, `wait_for_card` and returning `PAM_IGNORE`.

Reply:
Later, I've realized that the previous commit really introduces the `screen_savers` into the example config. So, you've right: we just need to join these commits. Done.